x-pack/plugins/apm/common/utils/to_kuery_filter_format.ts

```diff
@@ -5,10 +5,34 @@
  * 2.0.
  */
 
+import { isEmpty } from 'lodash';
+
+type Separator = 'OR' | 'AND';
+
 export function toKueryFilterFormat(
   key: string,
   values: string[],
-  separator: 'OR' | 'AND' = 'OR'
+  separator: Separator = 'OR'
 ) {
   return values.map((value) => `${key} : "${value}"`).join(` ${separator} `);
 }
+export function mergeKueries(
+  kuery1: string,
+  kuery2: string,
```

Comment:
Small suggestion, we could make this function take an array of queries, and then the logic inside could be boiled down to `.filter(...).join(...)`.